ckan/controllers/group.py

```diff
@@ -314,8 +314,14 @@ def pager_url(q=None, page=None):
 
             facets = OrderedDict()
 
-            default_facet_titles = {'organization': _('Organizations'),
-                                    'groups': _('Groups'),
+            org_type = h.default_group_type(u'organization') + u's'
+            org_label = h.humanize_entity_type(org_type).capitalize()
+
+            group_type = h.default_group_type(u'group') + u's'
+            group_label = h.humanize_entity_type(group_type).capitalize()
+
+            default_facet_titles = {'organization': _(org_label),
+                                    'groups': _(group_label),
```

Comment:
Here's my suggestion for the humanize_entity_type interface. This will let us cover all the common needs for customizing pluralizations, capitalizations etc in a plugin. 

```suggestion
            org_label = h.humanize_entity_type(
                entity_type=u'organization',
                object_type=h.default_group_type(u'organization'),
                purpose=u'facet label') or _('Organizations')
            group_label = h.humanize_entity_type(
                entity_type=u'group',
                object_type=h.default_group_type(u'group'),
                purpose=u'facet label') or _('Groups')

            default_facet_titles = {'organization': org_label,
                                    'groups': group_label,
```

Reply:
I left the gettext calls in above to handle the helper returning nothing and to support `extract_messages` use